tinygrad/runtime/ops_cuda.py

```diff
@@ -80,6 +80,10 @@ def __call__(self, global_size, local_size, *args, wait=False):
       end.synchronize()
       return start.time_till(end)*1e-3
 
+class PTXProgram(CUDAProgram):
```

Comment:
I don't think you need this, the renderer should just create a CUDAProgram with binary

Reply:
Definitely don't need both this and the above, but CUDAProgram gets that binary setting from runtime_args in ops. So I believe it either has to be set in ops or set in form in CUDAProgram. Could just add
`if getenv("PTX"): binary = True` in CUDAProgram's __init__ if you think that's cleaner? Or maybe rearrange the args for __init__ and wrap it with a `partial`, but I'm not sure if that binary arg is ever used outside of PTX?